src/components-examples/material/form-field/form-field-label/form-field-label-example.css

```diff
@@ -1,10 +1,9 @@
-.example-container {
-  display: flex;
-  flex-direction: column;
+.example-container .mat-form-field:nth-of-type(n+2) {
```

Comment:
This might be better as `.mat-form-field + .mat-form-field` so it doesn't have to depend on the number of form fields. The pattern seems to be used in a few more files in this PR.